hack/ci-deploy-release.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+#
+# Copyright (C) 2024 ScyllaDB
+#
+# This script deploys scylla-operator stack.
+# Usage: ${0} <operator_image_ref>
+# (Avoid using rolling tags.)
+
+set -euxEo pipefail
+shopt -s inherit_errexit
+
+source "$( dirname "${BASH_SOURCE[0]}" )/lib/kube.sh"
+
+if [[ -n "${1+x}" ]]; then
+    operator_image_ref="${1}"
+else
+    echo "Missing operator image ref.\nUsage: ${0} <operator_image_ref>" >&2 >/dev/null
+    exit 1
+fi
+
+source_raw="$( skopeo inspect --format='{{ index .Labels "org.opencontainers.image.source" }}' "docker://${operator_image_ref}" )"
+if [[ -z "${source_raw}" ]]; then
+    echo "Image '${operator_image_ref}' is missing source label" >&2 >/dev/null
+    exit 1
+fi
+source_url="${source_raw/"://github.com/"/"://raw.githubusercontent.com/"}"
+
+revision="$( skopeo inspect --format='{{ index .Labels "org.opencontainers.image.revision" }}' "docker://${operator_image_ref}" )"
+if [[ -z "${revision}" ]]; then
+    echo "Image '${operator_image_ref}' is missing revision label" >&2 >/dev/null
+    exit 1
+fi
+
+ARTIFACTS="${ARTIFACTS:-$( mktemp -d )}"
+
+kubectl_create -n=prometheus-operator -f="${source_url}/${revision}/examples/third-party/prometheus-operator.yaml"
+kubectl_create -n=haproxy-ingress -f="${source_url}/${revision}/examples/third-party/haproxy-ingress.yaml"
+
+kubectl_create -f="${source_url}/${revision}/examples/common/cert-manager.yaml"
+# Wait for cert-manager crd and webhooks
+kubectl wait --for condition=established --timeout=60s crd/certificates.cert-manager.io crd/issuers.cert-manager.io
+for d in cert-manager{,-cainjector,-webhook}; do
+    kubectl -n=cert-manager rollout status --timeout=5m "deployment.apps/${d}"
```

Comment:
missing wait-for-object-creation from ci-deploy: https://github.com/scylladb/scylla-operator/blob/b7ca9a4fce4aa060cec0bb8139382fdf8d909928/hack/ci-deploy.sh#L48. Is it superfluous there or accidentally missing here?

Reply:
should it be removed from the other script then? (not in this pr)

Reply:
I think so

Reply:
should we have a tracker for this? Even if it's in the backlog

Reply:
https://github.com/scylladb/scylla-operator/pull/1945